### DISCUSSION TITLE
Merging is blocked

### DISCUSSION THREAD

**Discussion by TypeOfFreak** [calculation-methods/M8O-303B-21#59] [Q&A]
Не могу замерджить PR, пишет, что я не авторизован. Во мне проблема или что мне с этим делать?
![Screenshot_2](https://github.com/calculation-methods/M8O-303B-21/assets/86594044/0859b648-af1a-4480-830d-30afd3914296)

**Comment by pivovarov-mai** [comment]
А вы авторизованы?

**Reply by TypeOfFreak** [reply]
Да...
